Tests/XCUITests/BookmarksTests.swift

```diff
@@ -210,6 +210,7 @@ class BookmarksTests: BaseTestCase {
         addNewBookmark()
         // Remove by swiping
         app.tables["Bookmarks List"].staticTexts["BBC"].swipeLeft()
+        mozWaitForElementToExist(app.buttons["Delete"])
```

Comment:
This test is failing in M1s because the swipe gesture does not work there :( we have to wrap this test so that it is run only on Intel. Please add a comment before the test definition as a reminder.
Thanks!

Reply:
Updated.

Reply:
This does not work for me... the swipe gesture is broken on M1, instead the bookmarked page is open..
What we can do is something like:
if processIsTranslatedStr() != m1Rosetta {

app.tables["Bookmarks List"].staticTexts["BBC"].swipeLeft()
            print(app.debugDescription)
            mozWaitForElementToExist(app.buttons["Delete"])
            app.buttons["Delete"].tap()
} else {
 // remove the bookmark using the Edit option and then the red button or just skipping the test
![Untitled](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/81777869-6280-4983-b7fa-8605a768ce30)

}

Reply:
Updated.